### ISSUE TITLE
Recurrence Rule for events not working

### ISSUE THREAD

**Issue by acollins1212** [acollins1212/sb2cal#1]
js_translation branch
I've narrowed this issue down to an invalid date in the $recurrence_string on scheduler.js. I've looked at RFC 5545 and it mentions that UNTIL dates should be in the same format as DTSTART. The issue might be that I DONT have DTSTART. Instead I'm using what the Google Calendar API reference uses: a "start" object.

**Comment by acollins1212** [comment]
once js_translation is merged into master, this issue will auto-close
